tests/test_risk_distributions.py

```diff
@@ -4,6 +4,34 @@
 
 from risk_distributions import risk_distributions
 
+distributions = [risk_distributions.Exponential, risk_distributions.Gamma, risk_distributions.Gumbel,
+                 risk_distributions.InverseGamma, risk_distributions.InverseWeibull, risk_distributions.LogLogistic,
+                 risk_distributions.LogNormal, risk_distributions.Normal, risk_distributions.Weibull]
+
+# FIXME: The following distributions break without updates to the processing functions
```

Comment:
should we explicitly skip those distributions that fail for now, in addition to those other tests ?

Reply:
What do you mean skip those distributions? Skip the test_cdf for those distributions? Because that's already happening (in a hacky way where I pulled those distributions out of the distributions list that gets parametrized over in the test_cdf.
I played around for a bit with trying to have pytest.mark.skipif use a condition based on the parameter (i.e., the distribution) but couldn't get it to work with either mark.parametrize or a fixture and I decided it wasn't worth spending a lot of time on it

Reply:
Oh I see. I fixated on the fixme :grin: